### DISCUSSION TITLE
Smart delivery flow

### DISCUSSION THREAD

**Discussion by gamer22026** [magicbell/community#177] [Questions] [answered]
Noticed that category overrides for Smart delivery only have selectable items from the default Smart Delivery for the project. For instance, if we want 99% of our categories to be just In-app, and 1% to be In-app, email, and SMS, then we have to setup the default Smart delivery to have In-app, email, and SMS, then we have to setup overrides for the 99% of our categories where we only want In-app. Would be much better to set default Smart delivery to just In-app (so the 99% of our categories are correct by default), and have the ability to add email and SMS on the category overrides. Is there a reason you can't select items in overrides unless they're already added to the default Smart delivery?

**Comment by unamashana** [comment, marked as answer]
@gamer22026 We will be improving this workflow in an upcoming release, and also make it configurable via an API endpoint you can use for deployments. Stay tuned!
